CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Changes
+
+* Change `Style/Documentation` to not require documentation for `ClassMethods` modules. ([@drenmi][])
```

Comment:
No direct editing to the CHANGELOG.md is required because existence of changelog/change_documentation_to_allow_class_methods.md :-)